content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -71,119 +82,114 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
 
     On the product page, you can see an error message on the **Book Reviews** section.
     The message reads: **"Ratings service is currently unavailable."**. The message appears because we
-    switched to use the `v2` subset of the `ratings` service without deploying the MongoDB service.
+    now use the `v2` subset of the `ratings` workload but we haven't deployed the MongoDB workload.
 
-1. Deploy the MongoDB service:
+1. Deploy the MongoDB workload:
 
-    * To deploy MongoDB in a cluster **with** automatic sidecar injection enabled:
+    {{< tabset cookie-name="sidecar" >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
-        {{< /text >}}
+    {{< tab name="With automatic sidecar injection" cookie-value="auto" >}}
 
-    * To deploy MongoDB in a cluster **without** automatic sidecar injection enabled:
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
+    {{< /text >}}
+
+    {{< /tab >}}
+
+    {{< tab name="With manual sidecar injection" cookie-value="manual" >}}
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
+    {{< /text >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
-        {{< /text >}}
+    {{< /tab >}}
+
+    {{< /tabset >}}
 
 1. Go to the Bookinfo product page at `http://$GATEWAY_URL/productpage`.
 
 1. Verify that the **Book Reviews** section shows the reviews.
 
-## Enabling Istio authorization
+1. Apply a default `deny-all` policy for the MongoDB workload:
```

Comment:
This line should come right before the command. The explanation (lines 115-117) should come before it.

Reply:
See my response in https://github.com/istio/istio.io/pull/5196/files#r340320873, if we do so, the page looks just strange because the text will look like they belong the previous numbered item, which is incorrect.